New version: TidierPlots v0.5.5

JuliaRegistrator opened the pull request:
- Registering package: TidierPlots
- Repository: https://github.com/TidierOrg/TidierPlots.jl
- Created by: @rdboyes
- Version: v0.5.5
- Commit: 67b2d670243bee7d04d62f4d1ee22db3959b4bed
- Reviewed by: @rdboyes
- Reference: https://github.com/TidierOrg/TidierPlots.jl/commit/67b2d670243bee7d04d62f4d1ee22db3959b4bed#commitcomment-138904160
- Description: Tidier data visualization in Julia, modeled after the ggplot2 R package.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6eee556faea1b69fdc1d53dd6e426062d2a2b18dfd4980bfe80343c4f2a1a8fdf85c2b6dfe1bb820b51fe637cb98f7474563aa1d92b4f747ea5c066ae15d6bfd5bfed8e743559542ed341b1c7b2541c22b44a32b9cb23a23fd1c8c058ecc922fcf4dc34c59b298a0a0e7015f41a2d14ff032459a252ed0d1e31d57cb2f37403ebc3a7dd58cf06c1ecd5614207819a0171919833e737e9ca940cb5b19c65853ce85f74c46f0ba4e4b5c85864dff0e1cb54 -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@rdboyes Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: TidierPlots v0.5.5

UUID: 337ecbd1-5042-4e2a-ae6f-ca776f97570a
Repo: https://github.com/TidierOrg/TidierPlots.jl.git
Tree: ab72b99fc415833c1523de9d13df330636acf05f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
